tests/Dockerfile

```
@@ -1,11 +1,9 @@
+# We use `latest` tag for tests proposes
+# hadolint ignore=DL3007
 FROM pre-commit-terraform:latest
 
-RUN apt update && \
-    apt install -y \
-        datamash \
-        time && \
-    # Cleanup
-    rm -rf /var/lib/apt/lists/*
```

Maybe not just drop, but adjust for alpine instead? 🤔 
```suggestion
    # Cleanup
    rm -rf /var/cache/apk/*
```

---

--no-cache

---

I meant a double check. Though, well, you're right, probably it makes no sense 👍🏻